ttkd-admin_panel/src/components/pages/Homepage.js

```diff
@@ -1,42 +1,20 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import logo from './btn_google_signin_light_normal_web@2x.png'
 import { Button, Form } from 'react-bootstrap';
+import { GoogleLogin } from 'react-google-login';
+
 
 class Homepage extends React.Component {
     constructor(props){
 		super(props);
-
-		this.state = {
-			fetching: true,
-			events: {}
-		}
-	}
-    render(){
-        return(
-            <div>
-                <Form>
-                    <Form.Group controlId="formBasicEmail">
-                        <Form.Label>Email address</Form.Label>
-                        <Form.Control type="email" placeholder="Enter email" />
-                        <Form.Text className="text-muted">
-                        We'll never share your email with anyone else.
-                        </Form.Text>
-                    </Form.Group>
-
-                    <Form.Group controlId="formBasicPassword">
-                        <Form.Label>Password</Form.Label>
-                        <Form.Control type="password" placeholder="Password" />
-                    </Form.Group>
-                    <Form.Group controlId="formBasicCheckbox">
-                        <Form.Check type="checkbox" label="Check me out" />
-                    </Form.Group>
-                    <Button variant="primary" type="submit">
-                        Submit
-                    </Button>
-                </Form>
-            </div>
-            );
     }
+ 
+  render(){
+    if (sessionStorage.getItem("email") !== null) {
```

Comment:
should we use the `checkLogin()` method here as well? It looks like it essentially does the same thing and will help DRY things out a bit